ss_dim: 2
maxStepSize: 2
goalTol: 2

